flake.nix

```diff
@@ -0,0 +1,111 @@
+{
+  description = "🎁 generate beautiful landing pages for your developer tools";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
...unstable?

Reply:
Despite the name, `nixos-unstable` is actually quite stable, though it tracks nixpkgs `master` (see [here](https://nixos.wiki/wiki/Nix_channels) for more info).